web/src/main/java/no/nav/foreldrepenger/fordel/web/app/rest/ManuellJournalføringRestTjeneste.java

```diff
@@ -0,0 +1,94 @@
+import no.nav.vedtak.sikkerhet.abac.beskyttet.ResourceType;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.MediaType;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+@RequestScoped
+public class ManuellJournalføringRestTjeneste {
+    private final ManuellJournalføringTjeneste manuellJournalføringTjeneste;
+    private final PersonInformasjon pdl;
+
+
+    @Inject
+    public ManuellJournalføringRestTjeneste(ManuellJournalføringTjeneste manuellJournalføringTjeneste,
+                                            PersonInformasjon pdl) {
+        this.manuellJournalføringTjeneste = manuellJournalføringTjeneste;
+        this.pdl = pdl;
+    }
+
+
+    @GET
+    @Path("/oppgaver")
+    @Operation(description = "Henter alle åpne journalføringsoppgaver for tema FOR og for saksbehandlers tilhørende enhet.", tags = "GOSYS", responses = {
+            @ApiResponse(responseCode = "500", description = "Feil i request", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = FeilDto.class))),
+            @ApiResponse(responseCode = "401", description = "Mangler token", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = FeilDto.class))),
+            @ApiResponse(responseCode = "403", description = "Mangler tilgang", content = @Content(mediaType = MediaType.APPLICATION_JSON, schema = @Schema(implementation = FeilDto.class)))
+    })
+    @BeskyttetRessurs(actionType = ActionType.READ, resourceType = ResourceType.FAGSAK)
+    public List<OppgaveDto> hentÅpneOppgaver() throws Exception {
+        var oppgaver = manuellJournalføringTjeneste.hentJournalføringsOppgaver();
+        List<OppgaveDto> journalføringOppgaver = new ArrayList<>();
+
+        if(oppgaver.isEmpty()) {
+            return Collections.emptyList();
+        }
+        oppgaver.forEach(oppgave -> journalføringOppgaver.add(lagOppgaveDto(oppgave)));
+        return journalføringOppgaver;
+    }
+
+    private OppgaveDto lagOppgaveDto(Oppgave oppgave) {
+        return new OppgaveDto(oppgave.id(), oppgave.journalpostId(), hentPersonIdent(oppgave.aktoerId()).orElse(null),
+                mapTema(oppgave.behandlingstema()), oppgave.fristFerdigstillelse(), mapPrioritet(oppgave.prioritet()), oppgave.beskrivelse(),
+                oppgave.opprettetTidspunkt(), harJournalpostMangler(oppgave));
+    }
+
+    private OppgavePrioritet mapPrioritet(Prioritet prioritet) {
+        return switch (prioritet) {
+            case HOY -> OppgavePrioritet.HOY;
+            case LAV -> OppgavePrioritet.LAV;
+            case NORM -> OppgavePrioritet.NORM;
+        };
+    }
+
+    private Optional<String> hentPersonIdent(String aktørId) {
+        if (!aktørId.isEmpty()) {
+            return pdl.hentPersonIdentForAktørId(aktørId);
```

Comment:
Trenger du denne? Her veksles aktørId -> FNR for en person. Mens OppgaveDto kaller den aktørId.
FNR (+navn) kan være grei for visning i GUI, men når du skal søke på saker i FPsak eller opprette sak så er aktørId like greit å sende som parameter.

Reply:
Mulig det kun er behov å vise fødselsnummer i vis oppgave. Det må vi ha en avklaring på.  Inntil videre sender jeg både aktørId og fødselsenummer slik at front end slipper å flippe fødselsnr til aktøriId